### DISCUSSION TITLE
极路由MT7621AT的CPU，速度只能跑到3M，CPU占用30%，请问是路由器的性能问题吗？ 我用电脑挂载直接飙到30M

### DISCUSSION THREAD

**Discussion by wagulu** [messense/aliyundrive-webdav#84] [Q&A] [answered]
如果是影响性能问题的话，考虑换硬件

**Comment by messense** [comment]
`cat /proc/cpuinfo` 看下。

**Comment by messense** [comment, marked as answer]
搜了一下这个 CPU 应该是个 MIPS 架构的双核，目前来说更推荐 arm CPU 架构的路由器吧，毕竟可以选择 64 位的（aarch64），而 MIPS 路由器据我所知常见的都是 32 位的。

**Reply by messense** [reply]
这里有个性能比较供参考：https://www.snbforums.com/threads/openssl-bench-for-fun-and-performance-client-and-server.54859/

![image](https://user-images.githubusercontent.com/1556054/137689514-7aa197a5-9c44-4e39-9546-7815cfdce1d2.png)

**Reply by wagulu** [reply]
谢谢， 晚点用IPQ807X试试能不能跑满
